src/components/Accordion/Accordion.tsx

```diff
@@ -118,8 +118,6 @@ class Accordion extends AutoControlledComponent<ReactProps<AccordionProps>, any>
   static Title = AccordionTitle
   static Content = AccordionContent
 
-  state: any = { activeIndex: [0] }
-
   getInitialAutoControlledState({ exclusive }) {
     return { activeIndex: exclusive ? -1 : [-1] }
```

Comment:
This line didn't work previously, that's why we have visual regressions there.

Reply:
To me it seems, according to the changes in screener that that line was indeed working:
![image](https://user-images.githubusercontent.com/4512430/50902375-01e57c00-141b-11e9-9fe1-5b2e3cc9490d.png)
 as the previous accepted screen has the first element inside active. Not quite sure how did you conclude that it was not working before.. Maybe I am missing something?

Reply:
I understand that this is part of the autocontroll props and should not be initialized, but can't understand why do you say it was not working previously